actors/builtin/storage_power/storage_power_actor_state.go

```diff
@@ -235,6 +256,54 @@ func (st *StoragePowerActorState) _getPledgeSlashForConsensusFault(currPledge ab
 	}
 }
 
+func asKeyer(a addr.Address) adt.Keyer {
+	return addrKeyWrapper{a}
+}
+
+type addrKeyWrapper struct {
+	addr.Address
+}
+
+func (kw addrKeyWrapper) Key() string {
+	return string(kw.Bytes())
+}
+
+func getStoragePower(rt vmr.Runtime, root cid.Cid, a addr.Address) (abi.StoragePower, bool) {
```

Comment:
returns a bool as some calls to this method make a decision if the address is not present in the hamt.